zanata-war/src/main/java/org/zanata/dao/ProjectDAO.java

```diff
@@ -433,4 +435,22 @@ public int getMaintainedProjectCount(HPerson maintainer, String filter) {
                         .setParameter("filter", "%" + sqlFilter + "%");
         return ((Long) q.uniqueResult()).intValue();
     }
+
+    public List<HProject> getProjects(EntityStatus... includeStatus) {
```

Comment:
Looking at this now, I think it would be better for readability to replace the one that takes some booleans with one that takes the list of `EntityStatus`, but I think there is a significant amount of code using the boolean one so it would not be a good change to make right now. Better to just keep the existing method for now. :+1: